src/libopensc/opensc.h

```diff
@@ -562,6 +562,13 @@ struct sc_reader_operations {
 /* Card (or card driver) supports key unwrapping operations */
 #define SC_CARD_CAP_UNWRAP_KEY			0x00001000
 
+/* Card (or card driver) can be initialized using C_InitToken() */
+#define SC_CARD_CAP_ENABLE_INITTOKEN			0x00002000
```

Comment:
If I understand correct, then this flag indicates whether the card can be initialized using the generic initialization method from here:
https://github.com/OpenSC/OpenSC/blob/e98f6733d9410b64249e7ca9028071a69ed8f004/src/pkcs11/framework-pkcs15.c#L1887-L1892
And this works if we have a card specific initializer in pkcs15init, right?
https://github.com/OpenSC/OpenSC/blob/e98f6733d9410b64249e7ca9028071a69ed8f004/src/pkcs15init/pkcs15-lib.c#L138-L171
Shouldn't we then use the latter list to determine the intialization capabilities rather than setting a flag in the generic driver, should we?